CHANGELOG.unreleased.md

```diff
@@ -13,6 +13,10 @@ Section headings should be at level 3 (e.g. `### Added`).
 
 ## Unreleased
 
+### Added
+
+- Added support for fetching artifact files via the artifact membership, i.e. with additional artifact collection membership context (@ibindlish in https://github.com/wandb/wandb/pull/9551)
```

Comment:
do you think this would be clear to our users? for me personally this sentence if very confusing, also you didn't break old behavior you just added new functionality if the server supports it, so i would mention that, and instead of i.e. just mention what is the new way you can fetch artifact files.